CollAction/Views/Projects/Create.cshtml

```diff
@@ -76,17 +76,22 @@
                         @Html.DropDownList("CategoryId", Model.Categories)
                         <span asp-validation-for="CategoryId" class="text-danger" />
                     </div>
-                    <div class="form-group">
+                    @*<div class="form-group">
```

Comment:
Hmm, same thing as before. I'd rather we just clean-up the front-end code, and not comment it out. The former code won't be useful to the API endpoint (and if it is, it'll still be in git).